static/install

```diff
@@ -104,104 +103,137 @@ tar -xf "$tmp_tar" -C "$tmp_dir" > /dev/null 2>&1 ||
 mv "$tmp_exe" "$exe" ||
     error "Failed to move executable from $tmp_exe to $exe"
 
+rm -rf "$install_dir/env"
+mv "$tmp_tar_dir/env" "$install_dir" ||
+    error "Failed to move env from $tmp_tar_dir to $install_dir"
+
+rm -rf "$install_dir/completions"
```

Comment:
I would be in favour of "defensive programming"... I think we should execute `rm -rf` only if we detect the directory. IMO, the less we use `rm -rf` the safest it is (even if here it is for a small and targeted think).

Reply:
Agreed. Fixed.